experiment/measurer.py

```diff
@@ -92,6 +92,34 @@ def get_coverage_infomation(coverage_summary_file):
         return json.loads(summary.readlines()[-1])
 
 
+def store_profdata_files(experiment_config: dict):
+    """Store profdata files to gcs bucket."""
+    logger.info('Start storing profdata files')
+    benchmarks = experiment_config['benchmarks'].split(',')
+    fuzzers = experiment_config['fuzzers'].split(',')
+    experiment = experiment_config['experiment']
+    for benchmark in benchmarks:
+        for fuzzer in fuzzers:
+            trial_ids = get_trial_ids(experiment, fuzzer, benchmark)
+            for trial_id in trial_ids:
+                store_profdata_file(fuzzer, benchmark, trial_id)
+    logger.info('Finished storing profdata files')
+
+
+def store_profdata_file(fuzzer, benchmark, trial_id):
```

Comment:
This function should be in coverage_utils, you shouldn't depend more on SnapshotMeasurer, but SnapshotMeasurer should depend on a coverage_utils Class or something (like inherit from it).